draft-ietf-rtcweb-jsep.xml

```diff
@@ -3349,10 +3349,23 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                         <t>Any "a=extmap" lines are parsed as specified
                         in <xref target="RFC5285" />, Section 5, and
                         their values are stored.</t>
-
-                        <t>TODO: identity, rtcp-rsize, rtcp-mux, and any
-                        other attributes valid at session level.</t>
-
+			
+			<t>Any "a=rtcp-rsize" lines are parsed as specified in
+			<xref target="RFC5506" />, Section 5, and their
+			values are stored.</t>
```

Comment:
These should be analogous to the text describing the handling of rsize, mux at the media level, which uses the following construction:
    If present, a single "a=rtcp-mux" attribute MUST be parsed as
    specified in [RFC5761], Section 5.1.1, and its presence or absence
    flagged and stored.

Suggest we also use the same ordering, i.e. mux, then rsize.